src/commands/deploy/buildImageInAzure/RunStep.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import type { DockerBuildRequest as AcrDockerBuildRequest } from "@azure/arm-containerregistry";
+import { AzExtFsExtra, AzureWizardExecuteStep } from "@microsoft/vscode-azext-utils";
+import * as path from 'path';
```

Comment:
I think that this is the only file still using path.

Reply:
Actually, don't worry about it. I don't think `dockerFilePath` is used to read any file which is the only reason we'd need to use Uris. We actually have a browser version of path, so `relative` should still work in this case.

If we ever wanted to implement our own relative in VSCode, here's the node implementation: https://github.com/nodejs/node/blob/main/lib/path.js#L491-L616

But doesn't feel worth it especially in this case.